Build/SaveLoadScreen.cc

```diff
@@ -1498,7 +1748,7 @@ void DoQuickLoad()
 
 bool AreThereAnySavedGameFiles()
 {
-	for (INT8 i = 0; i != NUM_SAVE_GAMES; ++i)
+	for (INT8 i = 0; i != (2 * NUM_SAVE_GAMES); ++i)
```

Comment:
All these 2s should either be a define similar to NUM_SAVE_GAME_BACKUPS or computed, if the way we track modes changes. Otherwise they'll be annoying to track down if anyone ever wants to add a third tab.